### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by genti94**
Created new project.

Created placeholder for the one-pager

replaced logo

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -6,95 +6,55 @@
...
-            {% static_placeholder "footer" %}
-            {% render_model_block request.current_page "changelist" %}
-                <ul class="list-inline">
-                    <li class="pull-right"><a href="#page-top" data-spy="scroll"><span class="fa fa-left fa-arrow-up"></span>{% trans "Back to top" %}</a></li>
-                    {% show_menu_below_id "footer" 0 0 100 100 "includes/menu/navigation.html" %}
-                </ul>
-            {% endrender_model_block %}
-        </div>
-    </footer>
+    <section class="section footer">
```
Comment (Brankub) [inline templates/base.html]: why this tag is empty?

File: templates/base.html [hunk 2]
```diff
@@ -6,95 +6,55 @@
...
+    </section>
 {% endblock footer %}
+
+ <div class="footer-bar">
+                <div class="container">
```
Comment (dhorelik) [inline templates/base.html]: indentation too large here

File: templates/fullwidth.html [hunk 3]
```diff
@@ -4,7 +4,66 @@
...
+        </div>
+    </section>
+
+    <section class="section portfolio">
+        <div class="container">
+            {% placeholder 'Portfolio' %}
+        </div>
+    </section>
+
+      <section class="section about">
```
Comment (vxsx) [inline templates/fullwidth.html]: weird intentation

File: templates/fullwidth.html [hunk 4]
```diff
@@ -4,7 +4,66 @@
...
+    <section class="section team">
+        <div class="container">
+            {% placeholder 'Team Members' %}
+        </div>
+    </section>
+
+    <section class="section blog">
+        <div class="container">
+            {% placeholder 'Blog' %}
+    </section>
```
Comment (vxsx) [inline templates/fullwidth.html]: unclosed div here

**Comment by dhorelik** [comment]
regarding `logo.png` - why the extra space on the right? It's transparent, so there might be something there I don't see. But seems like you should crop it more precisely

**Comment by Brankub** [comment]
from my side :+1: , let's wait for @vxsx and @dhorelik

**Comment by dhorelik** [comment]
apart from the uncropped image, lgtm

**Comment by vxsx** [comment]
code wise is ok

**Pull request merged**